sar/README.md

```diff
@@ -337,11 +339,5 @@ For those details please refer to the ["Additional properties" column in the tab
 - 1.6.1 / 1.7.1: `SourceDataRepository` and `RepositoryURL` are covered by STAC link structures. 
   All CARD4L compliant STAC Catalog are **required** to make intensive use of STAC link relation types such as
   `root`, `parent`, `child`, `item` and `collection`.
-- 1.6.9: `PeakSideLobeRatio` and `IntegratedSideLobeRatio` are *recommended* to be included in the resource linked to with the relation type
-  [`performance-indicators`](#stac-item-links).
-- 1.7.9: The CARD4L specification is not clear whether a string representation of a CRS is required
-  or whether an EPSG code would fullfil the requirement already.
 - 2.2: `raster:bands[*].values` is not standardized yet in STAC, this could change to `file:values`
   or something different with a similar structure in the future.
-- 4.4: In the CARD4L metadata specification the Gridding Convenvtion details seem to be required although not required in the textual specification.
```

Comment:
Has this been clarified by CEOS? I still see the ambiguity.

Reply:
It says "** (4.4) String content is a brief free text description of the gridding convention used (Threshold), provided as reference link if available (Target)" in the metadata specification, which is clear in my opinion and also reflects the other changes made in this PR. Or do you mean something else?

Reply:
The issue I have is that 4.4 in the PDF doesn't talk about any documentation requirements except that you need to use consistent gridding. But using something doesn't necessarily mean you need to document it. The requirement is only available once you look into the XLSX. So I'd like CEOS to clarify that first (the notes here are mostly a todo list for discussions/meetings with CEOS).

Reply:
I'll discuss 1.7.9 and 4.4 in the next meeting with CEOS (hopefully still in January) and then we can hopefully remove both of them in another PR.

Reply:
You're right, thanks for clarifying. I have also forwarded some comments to CEOS last month about ambiguities in the documents. I guess the more people like us look at these and point out problems, the better!